packages/web/pages/settings/integrations/notion.tsx

```diff
@@ -106,12 +106,17 @@ export default function Notion(): JSX.Element {
       <PageMetaData title="Notion" path="/integrations/notion" />
       <SettingsLayout>
         <VStack
+          distribution={'start'}
```

Comment:
Usually i will do `distribution="start"`

Reply:
Yeah, a copypasta I made 😂 

Changing it now